alf/config_util.py

```diff
@@ -311,7 +313,11 @@ def validate_pre_configs():
     """Validate that all the configs set through ``pre_config()`` are correctly bound."""
 
     if _PRE_CONFIGS:
-        raise ValueError("Cannot find config name %s" % _PRE_CONFIGS[0][0])
+        raise ValueError((
+            "A pre-config '%s' was not handled, either because its config name "
+            +
+            "was not found, or there was some conflict with existing configured "
```

Comment:
"conflict with existing configured" does not cause this.

Reply:
This valueerror was exactly what I saw for the issue causing this PR. When a pre_config cannot be configured, _PRE_CONFIG is not empty.